### PR TITLE
implement boilerplate frontend

### LINKED ISSUE #3: Boilerplate frontend

### PR DESCRIPTION
**Pull request by oliviagiandrea**
hi team! here's a first attempt at implementing some boilerplate frontend css. I basically created a global.css file, referenced it in the main page, and also imported our font family. for now, i've just styled a sidenav, a few buttons, some text, and an example form. 

looks like the package.json files were edited in the process though. let me know if that's an issue and i can revert. :)

desktop view: 
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,233 @@ TODO: description
...
+### `GET /api/insuranceCards?owner=USERNAME` - Get insurance cards by owner
+
+**Returns**
+
+- An array of insurance cards created by user with username `owner` in alphabetical order with respect to their purpose
+
+**Throws**
+
+- `400` if `owner` is not given
+- `404` if `owner` is not a recognized username of any user
```
Comment (wli-linda) [inline README.md]: Should there be another check for if the owner is the currently logged in user? Logic being that users should only be able to access their own info?

File: client/global.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
+:root {
```
Comment (jrdegreeff) [inline client/global.css]: Is there a particular reason why this is in it's own file rather than the <style> section of the App.vue vile?

**Review by jrdegreeff — CHANGES_REQUESTED** [review]
Looks good overall. See my minor question on structure. Would you also be able to add a basic layout (at least a sidebar, maybe a horizontal navbar as well but that isn't as urgent)? I will need this to have a place to add the user concept frontend for the alpha submission.

File: client/global.css (hunk 2 above)
Comment (oliviagiandrea) [reply client/global.css]: no reason - moved all global css to app.vue in latest push, as well as added a sidebar. 
for now, it's not very responsive and the links are obviously not linked to any real locations, but i'll write more css tomorrow when i'm home.

**Comment by jrdegreeff** [comment]
ah so it looks like you accidentally introduced a merge commit that mixed Isabella's changes back into this branch. Perhaps you have `pull` configured to auto merge instead of to auto rebase like I have. I'll try to fix this now because we are going to have a big mess combining this with the other PR otherwise. (This will not be a problem in the future -- it was just because we accidentally had an inconsistent state on the main branch when you started working.

**Comment by jrdegreeff** [comment]
ok I managed to linearlize the history and get rid of the unrelated changes. @oliviagiandrea please run `git rebase` before making any more changes or this will keep happening.

**Comment by jrdegreeff** [comment]
> desktop view: image

This updated version looks great to me! I'd say we can go ahead and merge. Did we decide what merge strategy we want to use?

**Review by jrdegreeff — APPROVED** [review]

**Pull request merged**